test/unittest/dds/publisher/CMakeLists.txt

```diff
@@ -387,9 +387,7 @@ if(QNX)
     target_link_libraries(DataWriterTests socket)
 endif()
 if(MSVC OR MSVC_IDE)
-    target_link_libraries(DataWriterTests ${PRIVACY}
-        iphlpapi Shlwapi
-        )
+    target_link_libraries(DataWriterTests iphlpapi Shlwapi)
```

Comment:
I think this is already done in line 381

Reply:
Done in 3344979f3